dist/tools/tunslip/tunslip6.c

```diff
@@ -147,18 +158,34 @@ stamptime(void)
 int
 is_sensible_string(const unsigned char *s, int len)
 {
-    int i;
+    if(len > 0) {
+        if (s[0] == 0x60) {
+            /* Possibly IPv6 packet with default traffic class, assume non-printable */
+            return 0;
+        }
+    }
 
-    for (i = 1; i < len; i++) {
+    int ret = 0;
+    for (int i = 1; i < len; i++) {
         if (s[i] == 0 || s[i] == '\r' || s[i] == '\n' || s[i] == '\t') {
```

Comment:
From my earlier experiences with SLIP and terminal multiplexing implementations: `\0` is just a good signal character to point out that something is printable: IPv4 packets start with `0x4` half-byte and IPv6 start with `0x6` half-byte. But I guess there are other reasons `\0` is included here.